scripts/import_cldr.py

```diff
@@ -422,6 +422,9 @@ def _process_local_datas(sup, srcdir, destdir, force=False, dump_json=False):
         parse_unit_patterns(data, tree)
         parse_date_fields(data, tree)
 
+        for elem in tree.findall('.//layout/orientation/characterOrder'):
```

Comment:
You should keep using parse wrapper methods! It's much easier to read.

Also, I was going to poke you about calling _should_skip_elem, but none of the elements in cldr 28 are draft, etc, so whatevs. :)